app/models/pusher.rb

```diff
@@ -80,7 +87,8 @@ def find
                                      platform: spec.original_platform.to_s,
                                      size: size,
                                      sha256: sha256,
-                                     pusher: user
+                                     pusher: user,
+                                     cert_chain: spec.cert_chain
```

Comment:
please add validation for max length of version.cert_chain.

Reply:
please add `to_s`. it would also be nice to add serialization for cert_chain so that the following code works:
```
version.cert_chain.each do |cert_pem|
...
end
```
https://api.rubyonrails.org/classes/ActiveRecord/Store.html